docs/source/pages/using.rst

```diff
@@ -203,30 +203,38 @@ This feature is enabled by default. To disable, or configure its behaviour in ot
 
 See :ref:`dump-defaults <action_dump_defaults>`, the CLI command to dump the default configurations in ``defaults.ini``. After making changes, see :ref:`analyze <analyze-action-cli>` CLI command for the option to pass the modified ``defaults.ini`` file.
 
-Within the configuration file under the ``repofinder.java`` header, five options exist: ``find_repos``, ``artifact_repositories``, ``repo_pom_paths``, ``find_parents``, ``artifact_ignore_list``. These options behave as follows:
+Within the configuration file under the ``repofinder.java`` header, three options exist: ``artifact_repositories``, ``repo_pom_paths``, ``find_parents``. These options behave as follows:
```

Comment:
I think the correct term to call `[[repofinder.java]]` is "section" rather than "header" (according to the [documentation of the `configparser` library](https://docs.python.org/3/library/configparser.html#quick-start)). 
```suggestion
Within the configuration file under the ``[[repofinder.java]]`` section,  three options exist: ``artifact_repositories``, ``repo_pom_paths``, ``find_parents``. These options behave as follows:
```

Reply:
It is essentially the same thing. The documentation you linked calls them "section headers", so either is fine really.

Reply:
I suppose if we want to be completely correct, more of the sentence would need to be adjusted as "under the ..." does not fit with "section".
Options are then:

1. "... under the <name> header..."
2. "...within the <name> section..."

Reply:
Hmm, that's very fair. Thanks, Ben. I think you can keep it as is.